### PR TITLE
PR for process Plugin (input parameters changes)

### PR DESCRIPTION
**Pull request by vitiwari**
Fixed Issues [TP-1961](https://boundary.jira.com/browse/PLUG-147) , [TP-1962](https://boundary.jira.com/browse/PLUG-148).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Displays CPU usage (%) & Memory usage (bytes) for specific processes. Uses regul
 
 |     OS    | Linux | Windows | SmartOS | OS X |
 |:----------|:-----:|:-------:|:-------:|:----:|
-| Supported |   v   |    -    |    -    |  -   |
+| Supported |   v   |    v    |    v    |  v   |
```
Comment (gokumar) [inline README.md:10]: Lets not add SmartOS here? since, we cant test it... or do we assume it will work?

File: param.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
 	[
 		{
 			"source" : "localhost",
-			"processArgs" : ".*meter",
+			"processParamOption" : "nameregex"
+      "processParamValue":".*meter",
```
Comment (gokumar) [inline param.json:7]: Invalid JSON.. plz fix

Comment (gokumar) [reply param.json:7]: missing ',' at the end of Line 6

File: README.md [hunk 3]
```diff
@@ -22,10 +22,10 @@ None
...
-|Process Name Regex|A regular expression to match the name of the process.                                                                                                                                                                                                         |
-|Process Path Regex|A regular expression to match the full path of the process.                                                                                                                                                                                                    |
-|Process CWD Regex |A regular expression to match the current working directory of the process.                                                                                                                                                                                    |
+|Source            |The source to display in the legend for the CPU data.|
+|Process Parameter Option|Parameter option for process matching. Parameter can be any of name/path/cwd/args regex. Choose the parameter option as Process Name Regex or Process Path Regex or Process CWD Regex or Process Args Regex |
```
Comment (gokumar) [inline README.md]: Change "Parameter option for process matching" to "Parameter option for matching process"

Comment (gokumar) [reply README.md]: Change "Choose the parameter option as Process Name Regex or Process Path Regex or Process CWD Regex or Process Args Regex" to
"Choose the parameter option as Process Name Regex, Process Path Regex, Process CWD Regex or Process Args Regex"

File: README.md (hunk 1 above)
Comment (vitiwari) [reply README.md:10]: We assume it should work as there is no interaction with underlying OS in this plugin, We query from meter so as long as meter supports the jsonrpc api on the platform, I hope it should be supported for the platform.

**Review by gokumar — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (vitiwari) [reply README.md]: Done ! please review .

File: param.json (hunk 2 above)
Comment (vitiwari) [reply param.json:7]: Fixed please review.

**Review by gokumar — APPROVED** [review]

**Comment by gokumar** [comment]
Looks good

**Pull request merged**